test/test_gather.py

```diff
@@ -0,0 +1,222 @@
+import os
+import subprocess
+
+
+def test_auto_gather_single():
+    # run example
+    os.system("python ../examples/log.py")
```

Comment:
Can you run this test without `os.system`? Like just creating one or more `LogManager`s directly>